boilerplate/bin/snackify.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Setup Script
+#
+echo "This will snackify your project and will delete some important folders like ios and android"
```

Comment:
If I were a user, I'd be scared off by this line and never want to run this command. Maybe we could add some documentation somewhere explaining why someone might want to run this script?

Reply:
I agree and updated this to a nice question and less scary

Reply:
I think if we could make this echo command longer, and list all the files that will be deleted, I would be satisfied as an end user to decided whether I want to continue or not

Something like:
```
In order to snackify this project, we must delete:
- /ios
- /android
- /e2e
- /test
- /app/services/reactotron
- jest.config.js
- metro.config.js
- webpack.config.js

Are you sure you want to continue?
```

Reply:
That's a great idea